petclinic-frontend/src/features/customers/api/updateOwner.ts

```diff
@@ -13,7 +13,5 @@ export const updateOwner = async (
 export const getOwner = async (
   userId: string
 ): Promise<AxiosResponse<OwnerResponseModel>> => {
-  return await axios.get<OwnerResponseModel>(
-    `http://localhost:8080/api/gateway/owners/${userId}`
-  );
+  return await axiosInstance.get<OwnerResponseModel>(`owners/${userId}`);
```

Comment:
Good job changing it to the axios instance so it redirects the user to the appropriate page